dockstore-webservice/src/main/resources/migrations.1.14.0.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<!--
+  ~    Copyright 2022 OICR and UCSC
+  ~
+  ~    Licensed under the Apache License, Version 2.0 (the "License");
+  ~    you may not use this file except in compliance with the License.
+  ~    You may obtain a copy of the License at
+  ~
+  ~        http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~    Unless required by applicable law or agreed to in writing, software
+  ~    distributed under the License is distributed on an "AS IS" BASIS,
+  ~    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~    See the License for the specific language governing permissions and
+  ~    limitations under the License.
+  -->
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd"
+                   context="1.14.0">
+    <changeSet author="ktran (generated)" id="addInvitationStatusColumn">
+        <addColumn tableName="organization_user">
+            <column defaultValue="PENDING" name="status" type="text">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+        <sql dbms="postgresql">
+            <comment>Migrate old organization user accepted column to the new invitation status column</comment>
+            UPDATE organization_user SET status='ACCEPTED' where accepted=true;
+            UPDATE organization_user SET status='PENDING' where accepted=false;
+        </sql>
+        <dropColumn columnName="accepted" tableName="organization_user"/>
```

Comment:
Once the migration is done, this will break old web service code that is still running. We often do have downtime with major releases, but we **could** try as an experiment to not have the downtime. So either

1. Can you please note this in the 1.14 deploy notes -- we always hypothesize if the web service will be broken by a release, we should start tracking it when we know it will.
2. Optionally, you could do something with a [trigger](https://stackoverflow.com/a/40901037), that keeps the new column in sync with the old column. Then in 1.15 we drop this column. Probably overkill, but OTOH if we want to eventually get to fully automated deploys, we may need to start doing this.

Reply:
I'm ok with either but second that this should either be clearly noted in the deploy notes or deprecated in the second case.

Reply:
Ended up going with number 2, maybe overkill, but it will be interesting to experiment with this